frontend/packages/component-library/src/accordion/faqAccordion/FAQAccordion.tsx

```diff
@@ -0,0 +1,66 @@
+import {ReactNode} from 'react';
+
+import Accordion, {AccordionProps} from './../Accordion';
+
+export type FAQItem = {
+  id: string;
+  label: string | ReactNode;
+  questionString: string;
+  content: string | ReactNode;
+  answerString: string;
+};
+
+export interface FAQAccordionProps extends AccordionProps {
+  /** List of FAQ items */
+  items: FAQItem[];
+  /** Custom className for additional styling */
+  className?: string;
+}
+
+/**
+ * ## Production-ready Checklist:
+ *  * (✔) implementation of component approved by design team;
+ *  * (✔) has storybook, covered with stories and documentation;
+ *  * (✔) has tests: test every prop, every state and every interaction that's js related;
+ *  * (see ./__tests__/FAQAccordrion.test.tsx)
+ *  * (✔) passes accessibility checks;
+ *
+ * ###  Status: ```Ready for dev```
+ *
+ * Design System: FAQ Accordion Component.
+ * Renders an Accordion with a list of items and JSON_LD script that's needed for Google SEO.
+ * (see https://developers.google.com/search/docs/appearance/structured-data/faqpage#json-ld)
+ */
+const FAQAccordion: React.FC<FAQAccordionProps> = ({
```

Comment:
Could we fold this into `Accordion` itself and use a union discriminator instead? For example:

```
<Accordion type={'faq'} />
```

Then the props for Accordion can be type discriminated:

```
interface AccordionProps {
  type: 'standard';
  items: Item[];
  // the standard stuff
}

interface FAQAccordionProps {
  type: 'faq';
  items: FAQItem[]
}
```

Accordion can then switch based on `type` whether or not to generate the ld json.

This would allow Accordion to be more generalized without needing another component for the JSON LD Functionality.

Reply:
If it should not be used elsewhere, would it make more sense to have this be a CMS component?

Reply:
I think it could be used on studio.code.org too.